usersync/syncer_test.go

```diff
@@ -300,31 +309,31 @@ func TestBuildTemplate(t *testing.T) {
 				URL:         "https://bidder.com/sync?redirect={{.RedirectURL}}",
 				RedirectURL: "{{xExternalURL}}",
 			},
-			expectedError: "template: anykey_usersync_url:1: function \"xExternalURL\" not defined",
+			expectedRendered: "https://bidder.com/sync?redirect=%7B%7BxExternalURL%7D%7D",
 		},
 		{
 			description: "Invalid Macro - Syncer Key",
 			givenSyncerEndpoint: config.SyncerEndpoint{
 				URL:         "https://bidder.com/sync?redirect={{.RedirectURL}}",
 				RedirectURL: "{{xSyncerKey}}",
 			},
-			expectedError: "template: anykey_usersync_url:1: function \"xSyncerKey\" not defined",
+			expectedRendered: "https://bidder.com/sync?redirect=%7B%7BxSyncerKey%7D%7D",
```

Comment:
Good catch. They used to be, but I've since updated the code to allow these and escape them instead of considering them macros. This was necessary since TappX uses a double curly brace in their user macro.

Reply:
Fixed.